### ISSUE TITLE
Remove all `;`

### ISSUE THREAD

**Issue by leobenkel** [leobenkel/PSF-LoginServer#13]

**Comment by Fate-JH** [comment]
Some examples of `;` that can definitely be removed are where if've shoved an otherwise multi-line `if else` into one line.  I may have one or two where I need to tell inspections "yes, this really is the end of the line".

The most common use of `;` is a standalone after a match case that does nothing. The semicolon is employed to mark that the case is blank, for future me to know that past me did not forget to write something.

**Comment by leobenkel** [comment]
yes i understand but there are better ways to do it. for the `if/else` you can just use `{...}` if more than one statement, it should be on 2 lines.
for blank statement the shorthand for returning a `Unit` is `()`
